releng/scripts/check_code_cleanliness.sh

```diff
@@ -12,6 +12,10 @@
 
 set -e
 
+if test ! -z "$(git status -s -uno)"; then
+    echo "You have changes. Please stash them before continuing."
+fi
```

Comment:
Right after submitting I wondered if this should actually `exit 1`?

Reply:
OOps. That somehow got lost. Sorry :disappointed:

Reply:
Actually not somehow. I removed it (and didn't comment it out :-( ) to be able to run the script on the other branch without committing it.